README.md

```diff
@@ -11,6 +9,9 @@ The project consists of three main parts:
 - Rest Gateway, that receive the lineage data from agent and stores it in the database
 - Web UI application that visualizes the stored data lineages
 
+![Spline diagram](https://user-images.githubusercontent.com/5530211/69061052-c4795400-0a18-11ea-9990-91836e2d648a.png)
```

Comment:
I see that you used my image 👍 I have the 'source code' for that image so in case you want to change anything in that I can still do it pretty easily.

Reply:
OK, I will create new version of the image.

Reply:
@leborgneromain could you please change the picture:
![Spline Schema v2](https://user-images.githubusercontent.com/5530211/70050339-fd93f580-15ce-11ea-88b2-4d79ee30d494.png)

The official name is actually _ArangoDB_ so I will use that name.